src/ImageSearchModal.jsx

```diff
@@ -212,7 +212,7 @@ export class ImageSearchModal extends React.Component {
                         <input className="form-control image-tag-entry"
                                id="image-search-tag"
                                type='text'
-                               placeholder={_("Tag")}
+                               placeholder={_("latest, 3.1, ...")}
```

Comment:
Wonder if it makes sense to translate this. Maybe for languages that do not use the same alphabet?

Reply:
The tags themselves aren't translated, right? Is "latest" the default tag?

Also: Should as have another example or two?

Reply:
> The tags themselves aren't translated, right?

Indeed, they would always be what someone tagged them. So if someone tags their container "najnovsi" (slovak for "latest") you would need to write "najnovsi". So I guess we don't want to translate this. I'll remove the translation.

> Is "latest" the default tag?

Yes, if you don't specify tag, it tries to pull `latest`. Should we then "placehold" just that, as that is what is going to happen if you don't change it.

> Also: Should as have another example or two?

I don't think that is necessary. If users know what tag is, they know which specific one they want. Otherwise latest is implied.

Reply:
> Should we then "placehold" just that, as that is what is going to happen if you don't change it.

Yeah, that could work. Having multiple shows that you can have numerical tags, but it does also sort of imply that commas and spaces are supported (for pulling down multiple tags?!)... and I'm pretty sure that's not the case.

Showing a value that is equivalent to the default state is a pattern that's commonly used.

Let's just go with an untranslated "latest", like you suggest.